icons/corejs/corejs-original.svg

```diff
@@ -0,0 +1 @@
03s2.542-2.401 5.65-5.51l5.652-5.65zM38.32 74.685l-.754-.758 2.926-2.93 1.519 1.51 7.185-7.187-9.41-9.41-17.31 17.308 9.411 9.412 7.187-7.187zm58.549-10.12-8.655-8.656-9.41 9.411 7.185 7.188 1.519-1.511 2.925 2.93-1.507 1.516 7.185 7.187 9.413-9.411zm-63.698-15.27-2.973-2.975-3.08 3.021c-1.694 1.662-5.613 5.53-8.708 8.596l-5.628 5.575 6.052 6.065 17.31-17.308zm76.42 8.643a3260.8 3260.8 0 0 0-8.708-8.597l-3.08-3.02-5.946 5.947 17.308 17.31 6.054-6.065zM45.934 30.905a394.65 394.65 0 0 1-4.212-1.081c-.106-.034-.659-.582-1.23-1.217a704.44 704.44 0 0 0-2.574-2.844c-.846-.928-1.537-1.765-1.537-1.86 0-.232.796-.952 1.053-.952.238 0 .222-.016 2.958 3.007 1.71 1.89 2.132 2.306 2.412 2.38.187.048 1.808.454 3.603.9s3.327.832 3.405.855c.26.077.367-.095.468-.76.056-.363.121-.748.146-.857.038-.17-.435-.52-3.311-2.46l-3.355-2.26-.001-5.747v-5.746h1.513l.008 10.64 2.746 1.845c1.51 1.015 2.775 1.855 2.81 1.867s.187-.298.337-.69c.593-1.544 2.003-3.25 3.522-4.261.646-.43 1.76-1.002 2.774-1.422.223-.093.209-.11-.36-.413-2.783-1.482-2.724-5.548.1-7.055l.69-.368 1.348-.033c.741-.018 1.348-.046 1.348-.062s-.148-.248-.328-.516c-.372-.552-.522-1.008-.523-1.581V9.82l-3.642-1.89-.05-5.22h1.229l.05 4.472 1.398.72c1.337.688 1.402.711 1.504.53.23-.409.988-1.03 1.628-1.333a4.945 4.945 0 0 1 4.278 0c.64.303 1.398.924 1.628 1.333.101.181.166.158 1.503-.53l1.398-.72.05-4.472h1.23l-.05 5.22-3.642 1.89v.394c-.001.573-.152 1.029-.523 1.58-.18.269-.328.501-.328.517s.606.044 1.348.062l1.347.033.69.368c2.825 1.507 2.883 5.573.1 7.055-.568.304-.583.32-.36.413 1.014.42 2.128.991 2.774 1.422 1.52 1.012 2.929 2.717 3.522 4.262.15.391.302.701.338.69.035-.013 1.3-.853 2.81-1.868l2.745-1.844.009-10.641h1.513v5.746l-.002 5.746-3.355 2.262c-2.877 1.938-3.35 2.29-3.31 2.459.024.109.089.494.144.857.102.665.208.837.469.76.078-.023 1.61-.407 3.405-.854s3.416-.853 3.603-.902c.28-.073.703-.49 2.412-2.379 2.736-3.023 2.72-3.007 2.959-3.007.257 0 1.053.72 1.053.953 0 .094-.692.931-1.537 1.86-.845.928-2.004 2.207-2.574 2.842-.674.75-1.137 1.186-1.324 1.245-.158.05-2.077.54-4.264 1.089l-3.977.997H64.757V20.019h-1.514v11.918l-13.289-.013zm7.779 25.529L43.474 46.17l2.316-4.87c1.274-2.68 2.317-4.919 2.318-4.977 0-.064-.954-.295-2.387-.577l-2.388-.47-7.328 7.298-.522-.516c-.286-.283-.521-.559-.521-.613 0-.187 7.703-7.806 7.892-7.806.158 0 5.229.965 5.794 1.102.178.044.253-.042.545-.62l.338-.671h2.883c1.956 0 2.884.032 2.884.1 0 .054-.638 2.584-1.419 5.622s-1.419 5.56-1.419 5.605c0 .121 1.322.443 1.415.344.043-.047.703-2.532 1.466-5.523a6246.6 6246.6 0 0 1 1.48-5.794l.092-.354h6.33v28.565h1.514V33.45h6.33l.092.354 1.48 5.794c.763 2.991 1.423 5.476 1.466 5.523.093.099 1.414-.223 1.414-.344 0-.045-.638-2.567-1.418-5.605s-1.42-5.568-1.42-5.623c0-.067.929-.1 2.885-.1h2.883l.338.672c.291.578.367.664.545.62.567-.138 5.636-1.102 5.795-1.102.189 0 7.89 7.62 7.89 7.807 0 .053-.234.329-.52.612l-.522.515-7.328-7.296-2.388.47c-1.434.281-2.388.512-2.387.576 0 .058 1.044 2.298 2.318 4.977l2.316 4.87-20.573 20.528z"/></svg>
```

Comment:
I think this solves the bug's bug. 😂

***Preview***
![test](https://user-images.githubusercontent.com/76992016/202858769-4022ff67-8d23-4658-8e74-4dc5dc935b2a.png)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="m56.053 2.71.049 5.22 3.642 1.89v.395c.001.573.153 1.029.524 1.58.18.269.328.5.328.516 0 .016-.608.044-1.35.062l-1.346.033-.69.367c-2.824 1.507-2.884 5.573-.1 7.055.567.304.584.321.36.414-1.013.42-2.129.991-2.775 1.422-1.52 1.012-2.928 2.717-3.521 4.262-.15.39-.302.7-.338.69-.035-.014-1.3-.853-2.81-1.868l-2.745-1.844-.01-10.64H43.76v5.746l.002 5.746 3.353 2.262c2.877 1.938 3.35 2.29 3.31 2.459-.023.109-.089.492-.144.855-.102.665-.207.839-.468.762-.079-.023-1.61-.409-3.405-.856-1.795-.447-3.416-.851-3.603-.9-.28-.073-.703-.492-2.412-2.381-2.736-3.023-2.72-3.006-2.96-3.006-.256 0-1.052.72-1.052.953 0 .094.692.93 1.537 1.86a685.26 685.26 0 0 1 2.574 2.841c.674.75 1.137 1.188 1.324 1.247.158.05 2.077.538 4.264 1.087l3.977.998h13.185V20.02h1.516v11.918h13.185l3.977-.999a310.904 310.904 0 0 0 4.264-1.087c.187-.06.65-.497 1.324-1.247.57-.635 1.729-1.913 2.574-2.841.845-.93 1.537-1.766 1.537-1.86 0-.233-.796-.953-1.053-.953-.239 0-.223-.017-2.959 3.006-1.709 1.889-2.132 2.308-2.412 2.38-.187.05-1.808.454-3.603.901-1.795.447-3.326.833-3.404.856-.261.077-.367-.097-.47-.762a20.32 20.32 0 0 0-.144-.855c-.04-.17.434-.521 3.31-2.46l3.354-2.261.002-5.746v-5.746H82.73l-.01 10.64-2.744 1.844a173.915 173.915 0 0 1-2.81 1.867c-.037.011-.189-.298-.339-.69-.593-1.544-2.001-3.249-3.521-4.26-.646-.432-1.762-1.003-2.776-1.423-.223-.093-.206-.11.362-.414 2.783-1.482 2.723-5.548-.102-7.055l-.69-.367-1.345-.033c-.742-.018-1.35-.046-1.35-.062s.148-.247.328-.516c.371-.551.523-1.007.524-1.58V9.82l3.642-1.89.05-5.22h-1.23l-.05 4.472-1.398.72c-1.337.688-1.401.71-1.502.53-.23-.41-.99-1.03-1.63-1.332A4.945 4.945 0 0 0 64 6.613a4.945 4.945 0 0 0-2.139.487c-.64.303-1.399.923-1.629 1.332-.1.18-.165.158-1.502-.53l-1.398-.72-.05-4.471h-1.23zM49.53 33.45l-.338.671c-.29.578-.367.663-.545.62-.567-.139-5.635-1.102-5.794-1.102-.19 0-7.891 7.62-7.891 7.806 0 .053.233.33.52.614l.523.513 7.326-7.295 2.389.47c1.434.28 2.387.511 2.386.575 0 .058-1.044 2.3-2.318 4.979l-2.314 4.869L64 66.65l20.525-20.48-2.314-4.87c-1.274-2.678-2.318-4.92-2.318-4.978-.001-.064.952-.295 2.386-.576l2.389-.469 7.326 7.295.524-.513c.286-.283.52-.56.52-.614 0-.187-7.703-7.806-7.892-7.806-.159 0-5.227.963-5.794 1.101-.178.044-.254-.04-.545-.619l-.338-.672h-2.883c-1.956 0-2.885.033-2.885.1 0 .055.64 2.585 1.42 5.623.78 3.038 1.418 5.56 1.418 5.605 0 .121-1.321.443-1.414.344-.043-.047-.704-2.532-1.467-5.523l-1.478-5.793-.092-.356h-6.33v28.567h-1.516V33.449h-6.33l-.092.356-1.478 5.793c-.763 2.99-1.424 5.476-1.467 5.523-.093.1-1.414-.223-1.414-.344 0-.045.638-2.567 1.418-5.605.78-3.038 1.42-5.568 1.42-5.623 0-.067-.929-.1-2.885-.1h-2.883zm-18.21 1.982L2.001 64.752l1.463 1.465 1.465 1.467 2.324-2.317 15.545 15.559c12.857 12.87 15.59 15.565 15.816 15.582.15.013 4.796-.245 10.322-.569l10.051-.59 1.985 1.99-15.604 15.604-1.408-1.406c-.774-.774-1.456-1.388-1.512-1.369-.14.048-6.433 4.658-6.44 4.717-.002.027 2.335 2.387 5.196 5.248 4.914 4.913 5.211 5.19 5.338 5.012.074-.103 1.155-1.57 2.404-3.262l2.272-3.074-1.436-1.444-1.433-1.441L64 100.274l15.65 15.65-1.433 1.441-1.436 1.444 2.272 3.074c1.249 1.692 2.33 3.159 2.404 3.262.127.178.424-.1 5.338-5.012 2.86-2.861 5.198-5.221 5.195-5.248-.006-.06-6.3-4.669-6.44-4.717-.055-.019-.737.595-1.51 1.37l-1.41 1.405L67.028 97.34l1.985-1.99 10.05.59c5.527.323 10.173.58 10.323.568.226-.017 2.96-2.712 15.816-15.582l15.545-15.559 2.324 2.317 1.465-1.467 1.463-1.465-29.318-29.32-2.934 2.931 1.16 1.158 1.158 1.159-31.12 31.12.603 10.263.601 10.26-.996 1.009c-.586.596-1.036.981-1.152.988-.116-.007-.566-.392-1.152-.988l-.996-1.01.601-10.26.604-10.261L31.936 40.68l1.158-1.159 1.16-1.158-2.934-2.931zM30.196 46.32l5.946 5.948-17.307 17.31-6.055-6.064 5.627-5.577a3260.8 3260.8 0 0 1 8.71-8.595l3.08-3.022zm67.606 0 3.08 3.022a3260.8 3260.8 0 0 1 8.709 8.596l5.627 5.576-6.055 6.064-17.307-17.31 5.946-5.948zm-58.018 9.588 9.41 9.412-7.183 7.188-1.52-1.512-2.926 2.932 1.508 1.515-7.185 7.188-9.412-9.412 8.654-8.655 8.654-8.656zm48.43 0 8.654 8.656 8.654 8.655-9.412 9.412-7.185-7.188 1.508-1.515-2.926-2.932-1.52 1.512-7.183-7.188 9.41-9.412zM52.885 69.01l4.885 4.885-.385 6.662c-.212 3.664-.387 6.724-.387 6.802 0 .078-2.54-2.4-5.648-5.51l-5.653-5.65 3.594-3.595 3.594-3.594zm22.23 0 3.594 3.594 3.594 3.595-5.653 5.65c-3.108 3.11-5.648 5.588-5.648 5.51 0-.078-.175-3.138-.387-6.802l-.385-6.662 4.885-4.885zm-32.35 10.12 5.653 5.651c5.176 5.173 5.633 5.655 5.393 5.686-.402.053-12.005.745-12.702.758l-.615.011-2.459-2.459-2.457-2.459 7.188-7.187zm42.47 0 7.187 7.188-2.457 2.46-2.46 2.458-.614-.011c-.697-.013-12.3-.705-12.702-.758-.24-.031.217-.513 5.393-5.686l5.652-5.65z"/></svg>

```